test/srt/run_suite.py

```diff
@@ -53,6 +53,9 @@
         "test_w8a8_quantization.py",
         "test_session_control.py",
         "test_fp8_kvcache.py",
+        "test_activation.py",
```

Comment:
Move all of them into a folder `test/srt/kernel`.
Later, we will organize the tests under `test/srt` with better structures.